tslint.json

```diff
@@ -5,7 +5,8 @@
   ],
   "rulesDirectory": [
     "tslint-consistent-codestyle",
-    "tslint-eslint-rules"
+    "tslint-eslint-rules",
+    "tslint-lines-between-class-members"
```

Comment:
@WoodNeck It looks related with coding style. In this case It would be better to discuss before applying it. 
Do you want it be coding style of egjs? Let's discuss here :)

Reply:
This is what I've said a long time ago, which is, adding a blank line between methods.
I think we've all agreed on it, but I was forgetting to add it for a while.
I've just remembered to add it on that commit.

Reply:
@WoodNeck I'm sorry I forgot it